openworld/sdk/generator/client/visitors/operations.py

```diff
@@ -41,7 +84,77 @@ def get_operations(parser: OpenAPIParser, model_path: Path) -> dict[str, object]
 
         sorted_operations.append(operation)
 
-    return {"operations": sorted_operations}
+    return sorted_operations
+
+
+def clean_non_schema_parameter_models(operations, non_schema_models, models_classnames_to_update):
+    for operation_index, operation in enumerate(operations):
+        for arg_index, arg in enumerate(operation.snake_case_arguments_list):
+            if arg.name == "body":
+                continue
+
+            for model_index, model in enumerate(non_schema_models):
+                # TODO: Do processing using a `DataType`` object instead of `type_hint`
+                if model.class_name in arg.type_hint:
+                    new_classname = update_classname_by_operation_id(operation.operationId, model.class_name)
+                    models_classnames_to_update[model.class_name] = new_classname
+
+                    operations[operation_index].snake_case_arguments_list[arg_index].type_hint = arg.type_hint.replace(model.class_name, new_classname)
+
+                    non_schema_models[model_index].class_name = new_classname
+                    break
+
+        operations[operation_index].snake_case_arguments_list = clean_unwanted_headers(operations[operation_index].snake_case_arguments_list)
+        operations[operation_index].snake_case_arguments = ", ".join(argument.argument for argument in operations[operation_index].snake_case_arguments_list)
+    return operations
+
+
+def update_non_schema_models_names(parser: OpenAPIParser, models_classnames_to_update):
+    for model_index, model in enumerate(parser.results):
```

Comment:
Use the `map` function.

Reply:
Can not use that due to dependency on outer data.